client/cmd_test.go

```diff
@@ -37,10 +37,10 @@ func TestValidateCmd(t *testing.T) {
 		args    []string
 		wantErr bool
 	}{
-		{"misspelled command", []string{"comission"}, true}, // nolint: misspell
+		{"misspelled command", []string{"commission"}, true}, // nolint: misspell
```

Comment:
I believe this was on purpose

Reply:
bump @alessio

Reply:
Ah yes I think so too. But the test is broken then, because it is still passing